src/reanimated2/hook/useScrollViewOffset.ts

```diff
@@ -89,37 +86,21 @@ function useScrollViewOffsetNative(
   ) as unknown as EventHandlerInternal<ReanimatedScrollEvent>;
 
   useEffect(() => {
-    // We need to make sure that listener for old animatedRef value is removed
-    if (scrollRef.current !== null && scrollRefTag.current !== null) {
-      eventHandler.workletEventHandler.unregisterFromEvents(
-        scrollRefTag.current
-      );
-    }
+    const elementTag = animatedRef?.getTag() ?? null;
```

Comment:
Now I notice that for the web version of the hook we use function `getWebScrollableElement` and for the native implementation we use `animatedRef.getTag`. Was there some particular reason for that? I'd be nice if `getTag` would work both on native and web.